st2common/st2common/models/db/__init__.py

```diff
@@ -199,6 +212,23 @@ def db_teardown():
     mongoengine.connection.disconnect()
 
 
+def db_cleanup(db_name, db_host, db_port, username=None, password=None,
+               ssl=False, ssl_keyfile=None, ssl_certfile=None,
+               ssl_cert_reqs=None, ssl_ca_certs=None, ssl_match_hostname=True):
+
+    connection = _db_connect(db_name, db_host, db_port, username=username,
+                             password=password, ssl=ssl, ssl_keyfile=ssl_keyfile,
+                             ssl_certfile=ssl_certfile,
+                             ssl_cert_reqs=ssl_cert_reqs, ssl_ca_certs=ssl_ca_certs,
+                             ssl_match_hostname=ssl_match_hostname)
+
+    LOG.info('Dropping database "%s" @ "%s:%s" as user "%s".',
+             db_name, db_host, db_port, str(username))
+
+    connection.drop_database(db_name)
+    return connection
```

Comment:
I do not like that this method is returning a connection object. It's a cleanup method :). The API seems less ideal. What does connection.drop_database(db_name) return? Do you want to just do return connection.drop_database(db_name)?

Reply:
It looks like connection.drop_database() is a PyMongo call that has no return statement.

Reply:
You are right. I looked into this and the method does not return anything. In this case, I'll make note of this as comment and change this method to not return anything.